modules/serverless-tls-ensuring-seamless-ca-rotation-eventing.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-config-tls-encryption-eventing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-tls-ensuring-seamless-ca-rotation-eventing_{context}"]
+= Ensuring seamless CA rotation
+
+Ensuring seamless CA rotation is essential to avoid service downtime or handle emergencies.
+
+.Prerequisites
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* You have installed the {ServerlessOperatorName}.
+* You have installed the {cert-manager-operator}.
+* You have installed the OpenShift (`oc`) CLI.
+
+.Procedure
+
+. Create a new CA certificate.
```

Comment:
```suggestion
. Create a CA certificate.
```
"Create a new" can be shortened to simply "Create", because when you create something, by definition, it is new.